fineract-provider/src/main/resources/application.properties

```diff
@@ -37,6 +37,7 @@ fineract.tenant.description=${FINERACT_DEFAULT_TENANTDB_DESCRIPTION:Default Demo
 
 fineract.mode.read-enabled=${FINERACT_MODE_READ_ENABLED:true}
 fineract.mode.write-enabled=${FINERACT_MODE_WRITE_ENABLED:true}
+fineract.mode.batch-enabled=${FINERACT_MODE_BATCH_ENABLED:true}
```

Comment:
There's no instance mode like this anymore, why was this required?

Reply:
@galovics This instance mode is used in two conditions - EnableFineractEventsCondition and EnableFineractEventListenerCondition

Reply:
@galovics : This particular mode is used in two conditions - EnableFineractEventListenerCondition and EnableFineractEventsCondition

Reply:
@galovics Please provide your feedback.